platforms/documentation/docs/src/docs/userguide/dep-man/declaring-depedencies/declaring_dependencies_basics.adoc

```diff
@@ -0,0 +1,221 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[declaring-dependencies-basics]]
+= Declaring Dependencies Basics
+
+[[sec:dependency-types]]
+== Types of dependencies
+
+There are three main types of dependencies in Gradle:
+
+1. *Module Dependencies*: Refer to libraries from external repositories.
+2. *Project Dependencies*: Refer to other projects in the same multi-project build.
+3. *File Dependencies*: Refer to local files or directories, such as `.jar` or `.aar` files.
+
+[[sub:module_dependencies]]
+=== 1. Module dependencies
+
+Module dependencies are the most common dependencies.
+They refer to a dependency that is identified by module coordinates (group, name, and version):
+
+====
+include::sample[dir="snippets/artifacts/externalDependencies/kotlin",files="build.gradle.kts[tags=module-dependencies]"]
+include::sample[dir="snippets/artifacts/externalDependencies/groovy",files="build.gradle[tags=module-dependencies]"]
+====
+
+Gradle offers multiple notations for declaring module dependencies, including string notation and map notation.
+
+- **String Notation:** Simplifies dependency declaration by combining the group, name, and version into a single string.
+- **Map Notation:** Allows for a more detailed specification, enabling the definition of all properties.
+
+For advanced configurations, such as making a dependency transitive, you can use closures alongside these notations.
+
+Refer to the link:{groovyDslPath}/org.gradle.api.artifacts.dsl.DependencyHandler.html[`DependencyHandler`] class in the API documentation for more examples and details.
+Additionally, explore the link:{javadocPath}/org/gradle/api/artifacts/ExternalModuleDependency.html[`ExternalModuleDependency`] class to understand the full range of properties and configuration methods available.
+
+[[sub:project_dependencies]]
+=== 2. Project dependencies
+
+Project dependencies allow you to reference other projects within a multi-project Gradle build.
+
+image::dependency-management-project-dependencies.png[]
+
+This is useful for organizing large projects into smaller, modular components:
+
+====
+include::sample[dir="snippets/dependencyManagement/declaringDependencies-projectDependencies/kotlin",files="web-service/build.gradle.kts[tags=project-dependencies]"]
+include::sample[dir="snippets/dependencyManagement/declaringDependencies-projectDependencies/groovy",files="web-service/build.gradle[tags=project-dependencies]"]
+====
+
+Gradle uses the link:{groovyDslPath}/org.gradle.api.Project.html#org.gradle.api.Project:project(java.lang.String)[`project()`] function to define a project dependency.
+This function takes the path to the target project within the build.
+The path is typically defined using a colon (`:`) to separate different levels of the project hierarchy.
```

Comment:
🤔 I wouldn't say project hierarchy - the hierarchical relationship between subprojects is kind of thin, and we want to encourage people to think about them independently, for Isolated Projects support.

Reply:
Updated wording.